src/audiomixerboard.cpp

```diff
@@ -1335,12 +1335,16 @@ void CAudioMixerBoard::ApplyNewConClientList ( CVector<CChannelInfo>& vecChanInf
 
 void CAudioMixerBoard::SetFaderLevel ( const int iChannelIdx, const int iValue )
 {
+    // Proposed change: if iChannelIdx is I_MY_CHANNEL and our own channel ID is a valid index
```

Comment:
```suggestion
    // If iChannelIdx is I_MY_CHANNEL and our own channel ID is a valid index
```

Reply:
Moved into header as a comment on the new function to return the real channel index.